test/fixtures/user/user.js

```diff
@@ -329,5 +329,19 @@ module.exports = () => {
       twitter_id: "ramsingh123",
       linkedin_id: "ramsingh123",
     },
+    {
+      username: "ankur",
+      first_name: "Ankur",
+      last_name: "Narkhede",
+      github_id: githubUserInfo[0].username,
+      github_display_name: githubUserInfo[0].displayName,
+      roles: {
+        member: true,
+        in_discord: true,
+      },
+      incompleteUserDetails: false,
+      updated_at: Date.now(),
+      created_at: Date.now(),
+    },
```

Comment:
Ankur is already the user in the array. can we rename the username to avoid confusion ?

Reply:
Done